system/rest.yaml

```diff
@@ -2198,3 +2198,56 @@ endpoints:
       path:
       - { type: string, name: lang,      required: true,  title: Language  }
       - { type: string, name: application, required: true, title: Application name }
+
+- title: Data Privacy Request
+  entrypoint: dataPrivacyRequest
+  path: "/data-privacy/requests"
+  apis:
+    - name: list
+      method: GET
+      title: List data privacy requests
+      path: "/"
+      parameters:
+        get:
+          - { name: query,              type: "string",     title: "Filter requests" }
+          - { name: kind,               type: "[]string",   title: "Filter by kind: correct, delete, export" }
+          - { name: status,             type: "[]string",   title: "Filter by status: pending, cancel, approve, reject" }
+          - { name: limit,              type: "uint",       title: "Limit" }
+          - { name: pageCursor,         type: "string",     title: "Page cursor" }
+          - { name: sort,               type: "string",     title: "Sort items" }
+    - name: create
+      method: POST
+      title: Create data privacy request
+      path: "/"
+      parameters:
+        post:
+          - { name: kind,         type: "string",   title: "Request Kind", required: true }
+    - name: update status
+      method: PUT
+      title: Update data privacy request status
+      path: "/{requestID}/status/{status}"
+      parameters:
+        path:
+          - { name: requestID,     type: "uint64",   title: "ID",             required: true }
+          - { name: status,        type: "string",   title: "Request Status", required: true }
+    - name: read
+      method: GET
+      title: Get details about specific request
+      path: "/{requestID}"
+      parameters:
+        path:
+          - { name: requestID, type: "uint64",     title: "Request ID", required: true }
+    - name: list responses
+      method: GET
+      title: List responses for a specific request
+      path: "/{requestID}/responses"
+      parameters:
+        path:
+          - { name: requestID, type: "string",     title: "Request ID", required: true }
+    - name: create response
+      method: GET
+      title: Create response for a specific request
+      path: "/{requestID}/responses"
+      parameters:
+        path:
+          - { name: requestID, type: "string",     title: "Request ID", required: true }
```

Comment:
`POST` with a payload?

Reply:
These routes were removed and no longer needed as we have new routes for request comments.